mbuild/utils/io.py

```diff
@@ -99,7 +99,20 @@ def import_(module):
     >>> # module from, etc) if the import fails
     >>> import tables
     >>> tables = import_('tables')
+
+    Notes
+    -----
+    The pybel/openbabel
     """
+    if module == 'pybel':
```

Comment:
Should this be an if/else statement?

Reply:
If the module we're trying to import is `pybel` and it can do so successfully, it will `return` out of the `if` clause. If not, it will fall back to the rest of the function where it will fail the `try` and then print out the pretty "try to install it from here" message. At least that's what I think should happen

Reply:
Ah yeah, I see what you mean now.